lib/rtoolsHCK.rb

```diff
@@ -134,7 +134,7 @@ def logger(level, progname = nil, &)
   #                    the local machine
   #                    (default: disabled)
   #   :l_script_file - The toolsHCK.ps1 file path on local machine
-  #                    (default: disabled)
+  #                    (default: shared/tools/toolsHCK.ps1)
```

Comment:
This needs an update.

I also suggest removing `l_script_file` option and hardcoding the path. We do not want to mix different toolsHCK and rtoolsHCK.